src/libs/actions/PersonalDetails.js

```diff
@@ -123,11 +132,15 @@ function fetch() {
             const allPersonalDetails = formatPersonalDetails(data.personalDetailsList);
             Onyx.merge(ONYXKEYS.PERSONAL_DETAILS, allPersonalDetails);
 
-            const myPersonalDetails = allPersonalDetails[currentUserEmail]
-                || {avatarURL: getAvatar(undefined, currentUserEmail)};
+            const currentUserPersonalDetails = allPersonalDetails[currentUserEmail]
+                || {avatar: getAvatar(undefined, currentUserEmail)};
+
+            // Add the first and last name to the current user's MY_PERSONAL_DETAILS key
+            currentUserPersonalDetails.firstName = data.personalDetailsList[currentUserEmail].firstName ?? '';
+            currentUserPersonalDetails.lastName = data.personalDetailsList[currentUserEmail].lastName ?? '';
```

Comment:
I'm not 100%, but it feels like some possibly unsafe accessing of object properties here. Would feel a bit better if we were using `lodashGet()` instead of the null coalesce which will throw `TypeError` the first time any of these are not objects (`??` won't safely evaluate to the right side like it does in PHP).